uk_bin_collection/uk_bin_collection/councils/StocktonOnTeesCouncil.py

```
@@ -35,12 +35,25 @@ def parse_data(self, page: str, **kwargs) -> dict:
+            # Dismiss cookie consent banner (clicks get intercepted otherwise)
+            try:
+                time.sleep(2)
+                cookie_btn = driver.find_element(
+                    By.XPATH,
+                    "//button[contains(translate(., 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',"
+                    "'abcdefghijklmnopqrstuvwxyz'),'accept')]",
+                )
+                driver.execute_script("arguments[0].click();", cookie_btn)
+                time.sleep(1)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)